modules/friendlytag.js

```diff
@@ -1671,6 +1680,9 @@ Twinkle.tag.callbacks = {
 				if (params.altLangTo) {
 					tagText += '|to=' + params.altLangTo;
 				}
+				if (params.doubleRedirectTarget) {
+					tagText += '|1=' + params.doubleRedirectTarget;
+				}
 			}
```

Comment:
This is an indent too deep; right now it's under `(tagName === 'R from alternative language')`, which obviously won't work!

What should happen is something like:

```suggestion
				if (params.altLangTo) {
					tagText += '|to=' + params.altLangTo;
				}
			} else if (tagName === 'R avoided double redirect' && params.doubleRedirectTarget) {
				tagText += '|1=' + params.doubleRedirectTarget;
			}
```